lib/govuk_index/presenters/expanded_links_presenter.rb

```diff
@@ -16,6 +16,10 @@ def organisation_content_ids
       content_ids("organisations")
     end
 
+    def parent_organisations
```

Comment:
nitpick

I wondered if this method should be `ordered_parent_organisations` to match the pattern on line 35 for `primary_publishing_organisation`